packages/atomic/src/components/facets/atomic-segmented-facet/atomic-segmented-facet.pcss

```diff
@@ -13,21 +13,24 @@
   border-radius: 0px var(--atomic-border-radius) var(--atomic-border-radius) 0px;
 }
 
+.box-container .value-box {
+  @apply hover:border-primary-light rounded-none p-2 border-2 flex truncate;
+  max-width: 20ch;
+}
+
```

Comment:
```suggestion
  @apply hover:border-primary-light rounded-none p-2 border-2 flex;
}

.box-container .value-label {
  max-width: 20ch;
}

```

Reply:
value label already has truncate & it should be the one that has a max num of strings